pkgs/top-level/python-packages.nix

```diff
@@ -8707,6 +8707,8 @@ in {
 
   winsspi = callPackage ../development/python-modules/winsspi { };
 
+  wlc = callPackage ../tools/misc/wlc { };
+
```

Comment:
if it's only meant to be a cli application. I would rather not add it as a python module.
```suggestion
```